app/(candidate)/dashboard/content/[slug]/components/ContentEditor.js

```diff
@@ -129,6 +130,18 @@ export default function ContentEditor({
     setIsEdited(true);
   };
 
+  const copyPlanCallBack = () => {
```

Comment:
We have a util component for copy to clipboard
app/shared/utils/CopyToClipboard.js
can you use it?

https://github.com/thegoodparty/gp-webapp/blob/develop/app/shared/utils/CopyToClipboard.js